zipline/pipeline/factors/statistical.py

```diff
@@ -27,13 +28,26 @@ def __new__(cls,
                 target_slice,
```

Comment:
What could this be renamed to? `target_slice` doesn't apply if it can be another 2D factor